base/typography/_links_config.scss

```diff
@@ -35,5 +36,13 @@ $active: '&:active';
       line-height: 1;
       margin-left: 2px;
     }
+  } @else if $fancy-before {
+    &:before {
+      content: "‹";
+      color: inherit;
+      font-style: normal;
+      line-height: 1;
+      margin-right: 2px;
+    }
```

Comment:
There's a lot of duplication here. What if we combine the shared properties into one ruleset?